### DISCUSSION TITLE
Having Trouble add PrimeVue to Nuxt

### DISCUSSION THREAD

**Discussion by SpaceRaider-Gonzo** [primefaces/community#885] [PrimeVue] [answered]
Tryed following your Guide but what ever i do it complains about: 
 importPT: { from: path.resolve(__dirname, './presets/lara/') },  missing an "as:"
And Could not resolve .... nuxt/primevue-plugin.mjs

`// https://nuxt.com/docs/api/configuration/nuxt-config
import path from 'path';

export default defineNuxtConfig({
  modules: [
    '@nuxtjs/tailwindcss',
    'nuxt-primevue'
  ],
  primevue: {
    importPT: { from: path.resolve(__dirname, './presets/lara/') }      //import and apply preset  
}
  
})`

![primeVue_import](https://github.com/primefaces/community/assets/151457040/15ec84c8-2236-4ca6-ae47-42f233872704)

![grafik](https://github.com/primefaces/community/assets/151457040/e5ed6604-a0e2-4117-92c9-abf67c9f4bf2)

**Comment by tugcekucukoglu** [comment]
Could you try to remove package-lock.json and node_modules. They might be outdated.

**Reply by SpaceRaider-Gonzo** [reply]
Its a complete fresh installation.
Removed package-lock.json
Removed node_modules folder
npm install
Folder and File are back
Still not working ->  WARN  Could not resolve .../primevue/.nuxt/primevue-plugin.mjs. 

Current Node Version: v20.8.1

**Comment by tugcekucukoglu** [comment]
Could you share your environment? Which version of primevue and nuxt-primevue are you using?

**Reply by SpaceRaider-Gonzo** [reply]
package.json
{
  "name": "nuxt-app",
  "private": true,
  "type": "module",
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@nuxtjs/tailwindcss": "^6.10.4",
    "nuxt": "^3.9.0",
    "nuxt-primevue": "^0.2.2",
    "vue": "^3.4.6",
    "vue-router": "^4.2.5"
  }
}

I tryed it step by step as in the Video:
https://www.youtube.com/watch?v=q5ck18NwYz8&t=407s

**Comment by jrojko** [comment]
Problem is in path.resolve
importPT: { from: '~/presets/lara/'} works for me

**Reply by justKMM** [reply]
yep worked for me! Thanks!!!

**Comment by seba-1911** [comment]
the error message says that the "as:" is missing. Try this. 

importPT: { from: path.resolve(__dirname, './presets/lara'), as: 'tailwind' }

**Reply by justKMM** [reply]
I tried this approach, and it said `Pre-transform error: Failed to resolve import "E:LearnProgTest-Nuxtpresetslara" from "virtual:nuxt:E:\LearnProg\Test-Nuxt\.nuxt\primevue-plugin.mjs". Does the file exist?`. Meanwhile the presets are already imported and put inside the directory. Could it be that Nuxt can't read the backslashes properly?

**Reply by justKMM** [reply]
ah nevermind it seems the problem child was path.resolve(). I resolved the issue without using the resolve function ;))

**Comment by mertsincan** [comment, marked as answer]
The main issue fixed in https://github.com/primefaces/primevue-nuxt-module/issues/43
